24_find_lab/findlab.txt

```diff
@@ -1,24 +1,33 @@
 1. Write command lines to find using "find":
 
   -. All files owned by yourself in /tmp folder
+	find /tmp -maxpath 1 -type f -user $(whoami)
 
   -. All files and directories not owned by you in /tmp folder
+	find /tmp \! -user $(whoami)
 
   -. All files, folders and links in your home folder whose name ends with a ".c"
+	find $HOME -L -name "*.c"
```

Comment:
You didn't forget to quote the pattern, that's good :) 

```-L``` is telling ```find``` to follow symbolic links, I'm not sure that that's what we want here... because if you happen to have a sym link to  ```/lib``` in your home dir, all the /lib/**/*.c files will show up...
```
vitali@Tux:~/tmp$ ln -s /lib link-to-lib
vitali@Tux:~/tmp$ ls -lh
total 0
lrwxrwxrwx 1 vitali vitali 4 ינו 13 12:24 link-to-lib -> /lib
vitali@Tux:~/tmp$ find ./ -name "*.c" | wc -l
0
vitali@Tux:~/tmp$ find -L ./ -name "*.c" | wc -l
371
```